.env-test

```diff
@@ -0,0 +1 @@
+PGDATABASE=nc_news_test
```

Comment:
These need be included in a `.gitignore` file so they aren't included in the repo 🕵️ 
The purpose of these files is that vulnerable info like database names aren't made available in the repo, so should be gitignored